### DISCUSSION TITLE
大佬们请教一下 EventSetter的Handler 无法绑定command

### DISCUSSION THREAD

**Discussion by X1a0Y** [BYJRK/DotNet-Discussions#62] [WPF 开发]
大佬们请教一下。我有一个listview想添加双击相应事件来响应我的command，当我尝试使用EventSetter时绑定了command以后显示事件Handler仅可绑定到委托类型Delegate的属性，无法正常运行。
![image](https://user-images.githubusercontent.com/121343212/226342206-ed5a3889-a855-438b-8e56-f94ef178c0e3.png)
其中的AddCommand是使用toolkit mvvm的RelayCommand生成器生成的

**Comment by eijjxx** [comment]
你看一下eventsetter的handler，这个是只能指向delegate的，如果要指向command，需要用behaviors:Interaction.Triggers
这玩意还不能直接在style里设置。需要用附加属性或者template来设置，挺麻烦的

**Reply by X1a0Y** [reply]
![image](https://user-images.githubusercontent.com/121343212/226567026-56b9236e-3e9b-43c1-a8d2-63d7030c0cf6.png)
jiejuel解决了 谢谢
